studio/LonaStudio/Documents/LogicDocument.swift

```diff
@@ -73,10 +73,6 @@ class LogicDocument: NSDocument {
 
     override func read(from data: Data, ofType typeName: String) throws {
         content = try LogicDocument.read(from: data)
-
-        if let url = fileURL {
-            LogicModule.invalidateCaches(url: url, newValue: program)
```

Comment:
this is already done when setting `content`

Reply:
Good catch